portfolio/src/main/webapp/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "webapp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "script.js",
```

Comment:
So as npm/node.js goes, I think this package.json is the default one, which means it's inaccurate for your project. For example, your portfolio site isn't called webapp, and script.js is not how you run the website. Also, I don't think you want to license your work under the ISC license.